perf/lua/1mops_write.lua

```diff
@@ -169,7 +169,7 @@ print(string.format([[
 
 box.cfg{
     log_level = 0,
-    listen = 0,
+    listen = '127.0.0.0:0',
```

Comment:
Why 127.0.0.0? Why not 127.0.0.1? The former is the loopback subnet AFAIK.

Reply:
Just a typo by misluck(